scripts/generateFiles.ts

```diff
@@ -424,10 +433,15 @@ module LanguageServer
       #
       {{/if}}
       class {{definition.interface.name}}
-        def initialize: ({{method_signature definition.allMembers}}) -> void
+        def initialize: ({{constructor_signature definition.allMembers}}) -> void
 
         @attributes: Hash[Symbol, untyped]
         attr_reader attributes: Hash[Symbol, untyped]
+        {{#each definition.allMembers}}
+
+        %a{pure}
```

Comment:
Steep consider methods with `pure` annotation as _pure_ (type narrowing works for the method).

Ref: https://github.com/soutaro/steep/wiki/Release-Note-1.1

Reply:
I hope RBS officially supports a way to declare this type of annotation some day😂